config/initializers/rack_profiler.rb

```diff
@@ -0,0 +1,6 @@
+if Rails.env == 'development'
```

Comment:
ah, sorry, i totally missed @jkeck's point about setting availability at config/initializer time.  totally fine w/ env var if settings are problematic on that front.